lib/client.js

```diff
@@ -69,9 +69,12 @@ client.prototype.request = function(method, path, params, callback){
         hostname: this.api_host,
         port: 443,
         method: method.toUpperCase(),
-        path: path,
+        path: path
     };
 
+    if(this.proxy_agent)
+        http_options["agent"] = this.proxy_agent;
```

Comment:
can you please update to add brackets for this if statement to keep consistency with the rest of the code.

```js
if(this.proxy_agent){
    http_options["agent"] = this.proxy_agent;
}
```